dwitter/templates/base.html

```diff
@@ -37,27 +37,36 @@
       // Check for compatibility
       if (window.localStorage){
         var switchTheme = function(){
-          // Change the imported stylesheet from main.css to main_dark.css or vice versa
-          // Change localstorage theme to 'dark'
+          // Shift themes
           var theme = localStorage.getItem('theme');
-          if (theme == 'dark') {
-            localStorage.setItem('theme','');
-          } else {
-            localStorage.setItem('theme','dark');
+          if (theme == '0') {
+            localStorage.setItem('theme','1');
+          } else if(theme == '1') {
+            localStorage.setItem('theme','2');
+          }
+          else {
+            localStorage.setItem('theme','0');
```

Comment:
Keep it simple:
```js
localStorage.setItem('theme', theme);
```

If they set an invalid value, we can deal with it at load time.